application/pubspec.yaml

```diff
@@ -28,6 +28,8 @@ environment:
 # the latest version available on pub.dev. To see which dependencies have newer
 # versions available, run `flutter pub outdated`.
 dependencies:
+  flutter_dotenv: ^5.1.0
+  supabase_flutter: ^2.0.0
   flutter:
     sdk: flutter
   http: ^0.13.3
```

Comment:
Make 

```
  flutter:
    sdk: flutter
```

always the first dependencie to mantain the most important at the top